fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadTask.java

```diff
@@ -965,6 +991,46 @@ public void afterCommitted(TransactionState txnState, boolean txnOperated) throw
         }
     }
 
+    public void collectProfile() {
+        RuntimeProfile profile = new RuntimeProfile("Load");
+        RuntimeProfile summaryProfile = new RuntimeProfile("Summary");
+        summaryProfile.addInfoString(ProfileManager.QUERY_ID, DebugUtil.printId(loadId));
+        summaryProfile.addInfoString(ProfileManager.START_TIME,
+                TimeUtils.longToTimeString(createTimeMs));
+
+        long currentTimestamp = System.currentTimeMillis();
+        long totalTimeMs = currentTimestamp - createTimeMs;
+        summaryProfile.addInfoString(ProfileManager.END_TIME, TimeUtils.longToTimeString(System.currentTimeMillis()));
+        summaryProfile.addInfoString(ProfileManager.TOTAL_TIME, DebugUtil.getPrettyStringMs(totalTimeMs));
+
+        summaryProfile.addInfoString(ProfileManager.QUERY_TYPE, "Load");
+        summaryProfile.addInfoString("StarRocks Version",
+                String.format("%s-%s", Version.STARROCKS_VERSION, Version.STARROCKS_COMMIT_HASH));
+        summaryProfile.addInfoString(ProfileManager.DEFAULT_DB, dbName);
+
+        Map<String, String> loadCounters = coord.getLoadCounters();
+        if (loadCounters != null && loadCounters.size() != 0) {
+            summaryProfile.addInfoString("NumRowsNormal", loadCounters.get(LoadEtlTask.DPP_NORMAL_ALL));
+            summaryProfile.addInfoString("NumLoadBytesTotal", loadCounters.get(LoadJob.LOADED_BYTES));
+            summaryProfile.addInfoString("NumRowsAbnormal", loadCounters.get(LoadEtlTask.DPP_ABNORMAL_ALL));
+            summaryProfile.addInfoString("numRowsUnselected", loadCounters.get(LoadJob.UNSELECTED_ROWS));
+        }
+
+        profile.addChild(summaryProfile);
+        if (coord.getQueryProfile() != null) {
+            profile.addChild(coord.getQueryProfile());
+            if (!isSyncStreamLoad()) {
+                coord.endProfile();
+                coord.mergeIsomorphicProfiles(null);
+            }
+        }
+
+        String profileContent = ProfileManager.getInstance().pushProfile(profile);
+        LOG.info(profileContent);
```

Comment:
It's not necessary to log the profileContent.

Reply:
ok